src/org/labkey/targetedms/SkylinePort/Irt/RetentionScoreCalculatorSpec.java

```diff
@@ -30,13 +30,17 @@
 
 public abstract class RetentionScoreCalculatorSpec implements IRetentionScoreCalculator
 {
-	public abstract Double ScoreSequence(String sequence);
+	@Override
+    public abstract Double ScoreSequence(String sequence);
```

Comment:
Formatting looks mangled. Maybe it was pre-mangled and we should do a general reformat in this PR too?